.gitignore

```diff
@@ -1,17 +1,22 @@
+# Configuration
+/app/config/parameters.yml
+
+# Libraries
+/bower_components
```

Comment:
We included both the bower components and the compiled sass files for the Fork backend in the repository for multiple reasons:

* easier for everyone to set up fork without the need of compilation/installation tools
* no issues during deployment (needing to bower install and compile sass)
* easier to create the zip download on the fork cms website